en/docs/learn/configuring-ws-trust-security-token-service.md

```diff
@@ -7,6 +7,11 @@ issuance. This API can be secured with the
 `         UserNameToken        ` or with any other WS-Security mechanism
 as explained below.
 
+!!! Tip "Before you begin"
+WS-Trust authentication is no longer supported by default in WSO2 IS 5.11.0 and has been introduced as a connector. 
+In order to use WS-Trust authentication, first you should download the connector from 
+[WSO2 Connector Store](https://store.wso2.com/store/assets/isconnector/list) and [install](https://github.com/wso2-extensions/identity-inbound-auth-sts/blob/master/docs/config.md#installing-the-connector) it.
```

Comment:
The content on the tip must be indented to be in line with where the word "Tip" starts..like this: 

```
!!! Tip "Before you begin"
     WS-Trust authentication is no.....
```

Reply:
done